analyzer-models/src/main/java/ffb/analyzer/models/espn/TransactionCounter.java

```diff
@@ -0,0 +1,66 @@
+package ffb.analyzer.models.espn;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Map;
+
+public class TransactionCounter extends EspnEntity<TransactionCounter> {
+    private float acquisitionBudgetSpent;
+
+    @JsonProperty("acquisitions")
+    private int waiverAcquisitions;
```

Comment:
`waiverAcquisitionCount` or something similar?

Reply:
Or `countOfAcquisitions` to keep things consistent.

Reply:
Lets do countOfAcquisitions and I'll update the IR count to be consistent as well.